testbot.py

```diff
@@ -160,4 +168,56 @@ async def join_game(arguments: list, message: discord.message):
     logging.info(f'Assigned {message.author} to {game_name}')
     await message.channel.send(f'Have fun playing {game_name}, {message.author}!')
 
+
+async def format_poll(message: discord.message):
+    command = message.content.replace("pb poll", '').strip()
+
+    if "\"" not in command or ',' not in command:
+        logging.debug('No poll title or arguments were passed')
+        await message.channel.send("You can't just make a poll without a title or arguments! Please insert a poll "
+                                   "title that is surrounded in \"\" as well as arguments proceeded with commas.")
+        return
+
+    title = command[command.find("\"") + 1:command.rfind("\"")]
+    options = command.replace(f'\"{title}\"', '').strip().split(", ")
+    logging.info(command)
+    poll = {
+        "title": title,
+        "options": options
+    }
+    logging.info('title --' + poll.get("title"))
```

Comment:
You don't need all these logs. Even if you need them, use `debug` for things only developers could make use of

Reply:
Forgot to remove those logs when I was debugging. They are removed now.